lib/runners/typescript.js

```diff
@@ -156,7 +156,11 @@ function prepareKarma(opts, interfaceType, runCode, fail) {
     },
   };
 
-  runKarma(opts, runCode, fail, interfaceType, config);
+  const nodeOpts = Object.assign({}, opts);
+  // run on the default node version
+  delete nodeOpts.languageVersion;
```

Comment:
Good catch. I was curious how long ago Node 2.4 was and apparently it doesn't even exist :D (it was io.js 2.4 released 2015-07-17).

If specifying the target for Karma is useful, we can do that too. I didn't touch Karma's configuration in https://github.com/Codewars/codewars-runner-cli/pull/476 because it was explicitly set to ES5.
https://github.com/Codewars/codewars-runner-cli/blob/aeb1d5d7628000217dc79b8a3d0594f9668d4f5d/lib/runners/typescript.js#L142-L147

Reply:
Yeah, I definitely thought about it. I couldn't imagine a good scenario for setting the Karma ES target. Nobody building websites today can safely target anything newer than ES5, and ES3 is too restrictive for most modern JS.

I'll leave it for something we can add in later if it become useful. As it is, we only support a single version of TS, and I'm not 100% sure how significantly the different target versions change the output in this case.